### PR TITLE
Add functional tests  for `fledge` config

### PR DESCRIPTION
**Pull request by marki1an**
Test for https://github.com/prebid/prebid-server-java/pull/2092

### CODE REVIEW

File: src/test/groovy/org/prebid/server/functional/model/bidder/Openx.groovy [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    String unit
+    String delDomain
+    String platform
+    Integer customFloor
+    Map customParams
+
+    static Openx getDefaultOpenx() {
+        new Openx().tap {
+            it.unit = PBSUtils.randomNumber
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/model/bidder/Openx.groovy:15]: Are all of those params mandatory?

File: src/test/groovy/org/prebid/server/functional/model/bidder/Openx.groovy [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import org.prebid.server.functional.util.PBSUtils
+
+class Openx implements BidderAdapter {
+
+    String unit
+    String delDomain
+    String platform
+    Integer customFloor
+    Map customParams
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/model/bidder/Openx.groovy:11]: It's a map of any type to any type or String to String?

File: src/test/groovy/org/prebid/server/functional/model/request/auction/ImpExt.groovy [hunk 3]
```diff
@@ -20,6 +20,7 @@ class ImpExt {
     ImpExtContextData data
     String tid
     String gpid
+    Integer ae
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/model/request/auction/ImpExt.groovy:23]: It's only OpenX specific? Or this param applies to PBS Core?

File: src/test/groovy/org/prebid/server/functional/model/response/auction/BidResponsePrebid.groovy [hunk 4]
```diff
@@ -10,4 +10,6 @@ class BidResponsePrebid {
 
     Long auctionTimeStamp
     Map passThrough
+    ExtBidResponseFledge fledge
+    List<SeatNonBid> seatNonBid
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/model/response/auction/BidResponsePrebid.groovy:14]: This shouldn't be in the PR. Why it's here? You're creating conflicts.

File: src/test/groovy/org/prebid/server/functional/model/response/auction/FledgeAuctionConfig.groovy [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import groovy.transform.ToString
+
+@ToString(includeNames = true, ignoreNulls = true)
+@JsonNaming(PropertyNamingStrategies.LowerCaseStrategy)
+class FledgeAuctionConfig {
+
+    String impId
+    String bidder
+    String adapter
+    Map config
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/model/response/auction/FledgeAuctionConfig.groovy:14]: It's a map of any type to any type or String to String?

File: src/test/groovy/org/prebid/server/functional/tests/BidderParamsSpec.groovy [hunk 6]
```diff
@@ -32,6 +32,7 @@ import static org.prebid.server.functional.model.request.auction.DistributionCha
 import static org.prebid.server.functional.model.request.auction.DistributionChannel.SITE
 import static org.prebid.server.functional.model.response.auction.ErrorType.PREBID
 import static org.prebid.server.functional.model.response.auction.MediaType.NATIVE
+import static org.prebid.server.functional.testcontainers.Dependencies.getNetworkServiceContainer
```
Comment (Net-burst) [inline src/test/groovy/org/prebid/server/functional/tests/BidderParamsSpec.groovy:35]: Looks like this shouldn't be here. If I remember correctly there is still a bug in IDEA that creates those imports. And they break the code...

**Review by Net-burst — CHANGES_REQUESTED** [review]

File: src/test/groovy/org/prebid/server/functional/model/response/auction/FledgeAuctionConfig.groovy (hunk 5 above)
Comment (marki1an) [reply src/test/groovy/org/prebid/server/functional/model/response/auction/FledgeAuctionConfig.groovy:14]: Any type to any type

File: src/test/groovy/org/prebid/server/functional/model/request/auction/ImpExt.groovy (hunk 3 above)
Comment (marki1an) [reply src/test/groovy/org/prebid/server/functional/model/request/auction/ImpExt.groovy:23]: Currently work only for OpenX but it's core

**Review by Net-burst — APPROVED** [review]
LGTM

**Pull request merged**